state/src/applicator/mod.rs

```diff
@@ -21,13 +24,24 @@ pub mod return_type;
 pub mod task_queue;
 pub mod wallet_index;
 
-// -------------
-// | Constants |
-// -------------
+// ---------------------
+// | Convenience Types |
+// ---------------------
 
 /// A type alias for the result type given by the applicator
 pub(crate) type Result<T> = std::result::Result<T, StateApplicatorError>;
 
+/// A struct packaging together all of the worker queues used by the state
+/// applicator
+pub struct ApplicatorQueues {
```

Comment:
we basically just instantiate this and immediately destruct it, i think we should just get rid of it.

Reply:
sure, just found myself agreeing with `clippy::too_many_arguments`, but idrc